content_scripts/courseRemover.js

```diff
@@ -28,14 +28,16 @@
      * Adds buttons that removes courses from the courses pane
      */
     async function addCourseRemovingOption() {
-        courses_list = document.getElementsByClassName('type_course depth_3 contains_branch');
+        var courses_list = document.getElementsByClassName('type_course depth_3 contains_branch');

```

Comment:
Better use querySelector

Reply:
Why? readability-wise?

Reply:
As I know querySelector is more used nowadays because it's more flexible
but it doesn't have much importance...